packages/react/src/lib/accessibility/Behaviors/Tree/treeItemBehavior.ts

```diff
@@ -47,4 +60,10 @@ export type TreeItemBehaviorProps = {
   open?: boolean
 }
 
+/** Checks if current tree item has a subtree and it is opened */
+const isSubtreeOpen = (props: TreeItemBehaviorProps): boolean => {
+  const { items, open } = props
+  return !!(items && items.length && open)
```

Comment:
Was it determined that checking `open` alone was not sufficient?

Reply:
yes, in `TreeItem` subtree is only rendered if `open` === `true` and `items` is not an empty array